<launch>
<arg name="filename" value="$(find dnn_experiment)/rosbag.bag"/>
<arg name="weight_file" value="$(find dnn_experiment)/edge_dataset/iternet.pth"/>
<arg name="depth" value="/cam0/helios2/depth/image_raw"/>

<node name="rosbag" pkg="rosbag" type="play"
    args="-r 0.5 --loop $(arg filename)" required="true" unless="$(eval filename=='')"/>

<node name="unet" pkg="dnn_experiment" type="ros_unet.py" output="screen" required="true">
    <remap from="~input_depth"  to="$(arg depth)" />
    <rosparam subst_value="True">
      weight_file: "$(arg weight_file)"
    </rosparam>
</node>

</launch>
